### PR TITLE
Consolidated branches

### PR DESCRIPTION
**Pull request by smithev95**
# Describe your changes
Integrated Tyler's tk-50 branch which translated the questionnaire by selecting language. Had to adjust a few areas where functionality broke due to discrepancies between this branch and main in things like function calls and page linking since tk-50 branched off an earlier build

   # Issue ticket number and link
   main and tk-50

   # Testing done
   Done manually, still needs more testing

   # Checklist before requesting a review
   [x] I have performed a self-review of my code
   [ ] If it is a core feature, I have added thorough tests.
   [x] Will this be part of a product update? If yes, please write one phrase about this update.

### CODE REVIEW

File: frontend/src/ClientDataForm.js
Comment (smithev95) [inline frontend/src/ClientDataForm.js:169]: This causes the button to skip the submit function, so it just links to `/selectlanguage` without calling `log_information`. This means no POST is made and no submission message pops up.

File: frontend/src/SelectQuestionPage.js
Comment (ployniti) [inline frontend/src/SelectQuestionPage.js:10]: Is the `useContext` and `LanguageContext` needed here?
If not, I would remove the unused imports.

File: frontend/src/AdminPanel.js
Comment (ployniti) [inline frontend/src/AdminPanel.js:1]: This file is serving the same purpose as `AdminPage.js` at the `/admin` path in `App.js`.
I don't think we should merge this file in this PR, especially if we're going to go with the navbar look in `AdminPage.js`

File: frontend/src/App.js
Comment (ployniti) [inline frontend/src/App.js:10]: Remove `AdminPanel` import if we decide to not merge `AdminPanel.js`.
Refer to the comment on the file above.

File: frontend/src/App.js
Comment (ployniti) [inline frontend/src/App.js:30]: Remove `<Route path="/adminpanel" element={<AdminPanel />} />` if `AdminPangel.js` is not merged.

File: frontend/src/ClientDataForm.js
Comment (ployniti) [inline frontend/src/ClientDataForm.js:5]: Remove `Link` import because it's not used.

File: frontend/src/ClientDataForm.js
Comment (ployniti) [inline frontend/src/ClientDataForm.js:64]: Do we need the `if` statement here?
Looks like whether `result` is True or not, we wanted to `window.location.href='/selectlanguage';` anyways so I think we shouldn't need an `if` statement here.

File: frontend/src/ClientDataForm.js
Comment (ployniti) [inline frontend/src/ClientDataForm.js:129]: We should look into making this spinner it's own component so it can be used on other pages while loading.
I added an issue on our to-do.

File: frontend/src/UpdateLanguages.js
Comment (ployniti) [inline frontend/src/UpdateLanguages.js:1]: I wouldn't merge this file with this PR.
I think it'd be best to separate this adding/updating languages feature into a different branch after merging other changes on this PR since it's not completely working right now and this PR is getting big.

**Review by ployniti — CHANGES_REQUESTED** [review]
Great work on consolidating changes and fixing the form submission from translated questions!

I left comments and change requests. Let me know if y'all have questions.

**Pull request merged**
